Makefile

```diff
@@ -13,34 +13,38 @@
 # the Doc Template for RISC-V Extensions.
 
 DATE ?= $(shell date +%Y-%m-%d)
-VERSION ?= v1.0.0
-REVMARK ?= Draft
 DOCKER_RUN := docker run --rm -v ${PWD}:/build -w /build \
 riscvintl/riscv-docs-base-container-image:latest
 
+# DO NOT SPECIFY A VERSION OR REVMARK HERE! The revision version and mark
+# (draft/release/etc) should be marked via the :revnumber: and :revremark: fields
+# in the source document itself.
 SRC_DIR := ./src
 BUILD_DIR := build
 HEADER_SOURCE := $(SRC_DIR)/docs-dev-guide.adoc
+GIT_SHA_DOC := $(SRC_DIR)/git_sha.adoc
 
 ASCIIDOCTOR_PDF := asciidoctor-pdf
 ASCIIDOCTOR_HTML := asciidoctor
 OPTIONS := --trace \
            -a compress \
            -a mathematical-format=svg \
-           -a revnumber=${VERSION} \
-           -a revremark=${REVMARK} \
-           -a revdate=${DATE} \
            -a pdf-fontsdir=docs-resources/fonts \
            -a pdf-theme=docs-resources/themes/riscv-pdf.yml \
            -D $(BUILD_DIR) \
-           --failure-level=ERROR
+           --failure-level=WARNING
```

Comment:
Haven't tried to grok the entirety of this PR yet, but I strongly endorse treating warnings as fatal.